### PR TITLE
LPD-34345 Update hyperlink in email notification

### PR DESCRIPTION
**Pull request by brookedalton**
https://liferay.atlassian.net/browse/LPD-34345

### CODE REVIEW

**Comment by brookedalton** [comment]
ci:test:sf

**Comment by brookedalton** [comment]
https://github.com/user-attachments/assets/0d021aae-2d50-4cea-aab3-a265492a8926

**Comment by brookedalton** [comment]
ci:test:publications

File: modules/apps/change-tracking/change-tracking-web/src/main/java/com/liferay/change/tracking/web/internal/portlet/action/InviteUsersMVCResourceCommand.java [hunk 1]
```diff
@@ -299,7 +300,8 @@ private Role _getRole(int roleValue, ThemeDisplay themeDisplay)
 	}
 
 	private void _sendEmail(
-			long ctCollectionId, long receiverUserId, ThemeDisplay themeDisplay)
+			CTCollection ctCollection, long ctCollectionId, long receiverUserId,
+			ThemeDisplay themeDisplay)
```
Comment (pei-jung) [inline modules/apps/change-tracking/change-tracking-web/src/main/java/com/liferay/change/tracking/web/internal/portlet/action/InviteUsersMVCResourceCommand.java:304]: We don't need both `ctCollection` and `ctCollectionId` in the params. We can get the object from the id or you can pass in the object if you don't want to do another fetch.

**Review by pei-jung — CHANGES_REQUESTED** [review]

**Comment by brookedalton** [comment]
ci:test:sf

**Review by pei-jung — APPROVED** [review]

**Comment by pei-jung** [comment]
ci:forward

**Comment by brookedalton** [comment]
ci:forward:force

**Pull request closed without merging**